enhancements/network/ovn-hypershift-live-migration.md

```diff
@@ -0,0 +1,319 @@
+3. A point to point routing is used so one node's subnet IP can be routed from different node
+3. The VM's gateway IP and MAC are independent of the node they are running on using proxy arp
+4. OVN missing ARP proxy features, configure MAC and IPv6, will be part of future 23.06 release, this is the [commit](https://github.com/ovn-org/ovn/commit/77846b215f317695384bd4bd27a647f6607413b1)
+
+
+### API Extensions
+
+n/a
+
+## Design Details
+
+### OVN-Kubernetes 
+
+#### Topology
+
+**Point to point routing:**
+
+When the VM is running at a node that do not "own" it's IP address, like what happend
+after a live migration, do point to point routing with a policy for outbound traffic and a static route for
+inbound. By doing this, the VM can live migrate to different node and keep previous 
+addresses (IP / MAC), thus preserving n/s and e/w communication, and ensuring traffic
+goes over the node where the VM is running. The latter reduces inter node communication.
+
+If the VM is going back to the node that "owns" the ip those static routes and 
+policies should not be configured.
+
+```text
+       ┌───────────────────────────┐
+       │     ovn_cluster_router    │
+┌───────────────────────────┐   ┌────────────────────────────┐
+│ static route              │───│ policy                     │
+│ prefix: 10.244.0.8        │   │ match: ip4.src==10.244.0.8 │
+│ nexthop: 10.244.0.8       │   │ action: reroute            │
+│ output-port: rtos-node1   │   │ nexthop: 10.64.0.2         │
+│ policy: dst-ip            │   │                            │  
+└───────────────────────────┘   └────────────────────────────┘ 
+```
+**Nodes logical switch ports:**
+
+The "router" logical switch port per node need to activate "arp_proxy" option
+with a fixed MAC and VMs gateway so VM neighbor cache is consistent 
+after live migration also VM's subnet need to be added so VM can ping IP's 
+from same subnet after live migration.
+
+```text
+    ┌────────────────────┐   ┌────────────────────┐
+    │logical switch node1│   │logical switch node2│
+    └────────────────────┘   └────────────────────┘
+┌────────────────────┐  │     │   ┌─────────────────────┐      
+│ lsp stor-node1     │──┘     └───│ lsp stor-node2      │      
+│ options:           │            │ options:            │    
+│  arp_proxy:        │            │   arp_proxy:        │
+│   0a:58:0a:f3:00:00│            │    0a:58:0a:f3:00:00│
+│   169.254.1.1      │            │    169.254.1.1      │
+│   10.244.0.0/24    │            │    10.244.0.0/24    │
+└────────────────────┘            └─────────────────────┘
+```
+
+**VMs logical switch ports:**
+
+The logical switch port for new VMs will use ovn-k ipam to reserve an IP address,
+and when live-migrating the VM, the LSP address will be re-used.
+
+CNI must avoid setting the IP address on the migration destination pod but ovn-k controllers should 
+continue book keeping it.
```

Comment:
```suggestion
CNI must avoid setting the IP address on the migration destination pod, but ovn-k controllers should 
preserve the IP allocation.
```

or something similar. The message is OK, but I find the wording a bit confusing. After all, you just want to say OVN-K must preserve the IP allocation, right ? Why not just say that ?